tmp/lanelet2_extension/include/lanelet2_extension/utility/query.hpp

```diff
@@ -134,6 +136,13 @@ std::vector<lanelet::NoParkingAreaConstPtr> noParkingAreas(const lanelet::ConstL
  */
 std::vector<lanelet::SpeedBumpConstPtr> speedBumps(const lanelet::ConstLanelets & lanelets);
 
+/**
+ * [speedBumps extracts Crosswalk regulatory elements from lanelets]
```

Comment:
```suggestion
 * [crosswalks extracts Crosswalk regulatory elements from lanelets]
```

Reply:
fixed in https://github.com/autowarefoundation/autoware_common/pull/187/commits/c572b733aaab97f4cc00b8282714deb9be7d4408.